go-controller/pkg/ovn/egressgw_test.go

```diff
@@ -2215,8 +2215,9 @@ var _ = ginkgo.Describe("OVN Egress Gateway Operations", func() {
 								Networks: []string{"100.64.0.4/32"},
 							},
 							&nbdb.LogicalRouter{
-								Name: ovntypes.OVNClusterRouter,
-								UUID: ovntypes.OVNClusterRouter + "-UUID",
+								Name:  ovntypes.OVNClusterRouter,
+								UUID:  ovntypes.OVNClusterRouter + "-UUID",
+								Ports: []string{ovntypes.GWRouterToJoinSwitchPrefix + ovntypes.GWRouterPrefix + "node1" + "-UUID"},
```

Comment:
do you think we could do something similar to https://github.com/ovn-org/ovn-kubernetes/pull/4027/files#diff-67c56fe553627fead6230bca48b1ae6d32273b2c7b4606cedc08513be886b717 and add some db-entries generation functions to simplify this? (maybe in another PR?)

Reply:
I could do the same for almost all tests, and I agree there is improvements to be made in regards to verbosity of the tests. But I in general favor for the data to be in the test itself rather than hidden away in a function so I would have to think about something new. However, I just wanted to get the tests working without having to refactor them all which would be an un-assumable task mostly.